src/engine-actions/get-settled-swap-preimage.js

```diff
@@ -1,85 +1,41 @@
 const {
-  lookupInvoice,
-  subscribeInvoices
+  lookupInvoice
 } = require('../lnd-actions')
 
+const INVOICE_STATES = Object.freeze({
```

Comment:
[nit] add jsdoc

Reply:
I think the invoice states from the LND API are integers, I added a file to `constants` here that we could use: https://github.com/sparkswap/lnd-engine/pull/234/files#diff-6ee896676d47ea25651fd79124cce3c4R4

Reply:
Oh wait, I just noticed there is a configuration for strings, I'll have to change my PR https://github.com/sparkswap/lnd-engine/blob/master/src/utils/load-proto.js#L21